client/src/pages/private/SiteViewDetails.js

```diff
@@ -50,15 +55,24 @@ export default ({ match }) => {
   };
 
   const fetchDetails = useCallback(async () => {
-    const response = await fetch(`${API_URL}/api/v1/employer-sites/${id}`, {
-      headers: {
-        Authorization: `Bearer ${store.get('TOKEN')}`,
-      },
-      method: 'GET',
-    });
-
+    setIsLoading(true);
+    const response = await fetchSite(id);
     if (response.ok) {
-      setSite(await response.json());
+      const site = await response.json();
+      let phases = [];
+      if (featureFlag(flagKeys.FEATURE_PHASE_ALLOCATION)) {
+        phases = await fetchSitePhases(site.id);
+      }
+      const participants = await fetchSiteParticipants(columnIDs, site.siteId);
+      setIsLoading(false);
+      return setSite({
+        ...site,
+        hiredParticipants: participants.hiredParticipants,
+        withdrawnParticipants: participants.withdrawnParticipants,
+        phases,
+      });
+    } else {
+      return setSite({ hiredParticipants: [], withdrawnParticipants: [], phases: [] });
```

Comment:
Is there not a need to report any error? A lil toast? :)

Reply:
good call - I've updated it to use the same toast/error message as the PATCH.